### ISSUE TITLE
Inconsistencies in Vocab Referencing

### ISSUE THREAD

**Issue by SimonGruening** [exc-asia-and-europe/ziziphus#287]
There seem to be inconsistencies with the saving of the Getty ID in the Global Attributes. Looks like the Vocab, the ID and the source are sometimes saved and sometimes not.
![materialset_inconsistency](https://cloud.githubusercontent.com/assets/3437957/7497436/ac457f44-f41a-11e4-9959-d5f914fa8007.PNG)

**Comment by zwobit** [comment]
Mmmh i can not reproduce this, is this still valid? If so could you specify a record where it does not work?

**Comment by SimonGruening** [comment]
It's this WR http://kjc-sv016.kjc.uni-heidelberg.de:8080/exist/apps/tamboti/modules/search/index.html?search-field=ID&value=w_e690b96e-439c-4f37-85a5-2f10ac55bc34
It's in my private collection at the moment, but I added you as a user to this collection now. These are the VRA Sample Records I am currently working on. - And yes, the record still looks like this.

**Comment by MatthiasArnold** [comment]
i was not able to reproduce it on test and live servers. 
@SimonGruening can this be related to an older record? do you remember the sequence of data entering? did you add the attributes first? 
please try to reproduce the error and update this post. thanks!

cf records (created today):
productionserver: "Cluster Baustelle 2015-05-12_0849" (w_de3f9529-f84c-400d-8269-2c0ba154c42b) in collection "Baustelle"
http://kjc-sv016.kjc.uni-heidelberg.de:8080/exist/apps/tamboti/modules/search/index.html?search-field=ID&value=w_de3f9529-f84c-400d-8269-2c0ba154c42b

testserver: "Work record 2015-06-17 10.07.21.jpg" (w_44299b04-da45-42f9-8967-3c6e2db68e22) in collection "Matzes images"
http://kjc-ws2.kjc.uni-heidelberg.de:8650/exist/apps/tamboti/modules/search/index.html?search-field=ID&value=w_44299b04-da45-42f9-8967-3c6e2db68e22

nb: please provide direct links together with work titles and name of the collection for each record. note that searching for vra record id's currently does not work. thanks! (cf. http://kjc-sv013.kjc.uni-heidelberg.de/redmine/issues/1194)

**Comment by SimonGruening** [comment]
This was a newly added image from the CCO examples. I added the term first, attributes thereafter.
Couldn't reproduce the error anymore - neither in a new image, nor in the old one. I could delete "black chalk" - just the term -  and enter it anew and the vocab appeared. For "red chalk" I had to delete the hole subject term set - through the bin icon - and add a new term to make the vocab appear though.
